products/fairphone.md

```diff
@@ -10,6 +10,14 @@ releaseColumn: false
 releaseDateColumn: true
 
 releases:
+
+-   releaseCycle: "5"
+    highestAndroidVersion: "12"
```

Comment:
```suggestion
    highestAndroidVersion: "13"
```

According to https://support.fairphone.com/hc/en-us/articles/18020671537041-Fairphone-5-FAQ.